easybuild/easyconfigs/l/libunwind/libunwind-1.2.1-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'libunwind'
+version = '1.2.1'
+
+homepage = 'http://www.nongnu.org/libunwind/'
+description = """The primary goal of libunwind is to define a portable and efficient C programming interface
+ (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the
+ preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain
+ (non-local goto). The API supports both local (same-process) and remote (across-process) operation.
+ As such, the API is useful in a number of applications"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SAVANNAH_SOURCE]
+
+checksums = ['3f3ecb90e28cbe53fba7a4a27ccce7aad188d3210bb1964a923a731a27a75acb']
+
+builddependencies = [
+    # There is a bug in binutils 2.27 and later causing building of
+    # libunwind to fail
+    ('binutils', '2.26'),
+]
+
+dependencies = [
+    ('XZ', '5.2.3'),
+]
+
+preconfigopts = 'export LIBS="$LIBS -llzma" && '
```

Comment:
existing `libunwind` easyconfigs also have `export CFLAGS="$CFLAGS -fuse-ld=bfd"` in `preconfigopts`, no longer needed?

Reply:
What easyconfigs would that be? Can't find any in develop that has anything other than
preconfigopts = 'export LIBS="$LIBS -llzma" && '

Reply:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/l/libunwind/libunwind-1.2.1-GCCcore-6.4.0.eb#L24, which was added in #5542 (that's the only one though, but since it's also v1.2.1 it's highly relevant imho)

Reply:
Ah, must have forgotten that i wasn't in develop when i looked for it.